### PR TITLE
feat: consolidated search query

### PR DESCRIPTION
**Pull request by milorue**
Consolidated the search query into a single graphql query

Added wallet search back in

### CODE REVIEW

File: src/common/components/search/SearchBar.tsx [hunk 1]
```diff
@@ -48,12 +50,8 @@ const SearchBar: FC = () => {
     setShowResults(false);
   };
 
-  const [searchQuery, { data, loading, called }] = useMetadataSearchLazyQuery();
-
-  const [
-    profileSearchQuery,
-    { data: profileData, loading: profileLoading, called: profileCalled },
-  ] = useProfileSearchLazyQuery();
+  // const [searchQuery, { data, loading, called }] = useMetadataSearchLazyQuery();
```
Comment (austinmilt) [inline src/common/components/search/SearchBar.tsx:53]: delete commented code

File: src/common/components/search/SearchBar.tsx [hunk 2]
```diff
@@ -143,16 +137,17 @@ const SearchBar: FC = () => {
...
               <LoadingSearchItem />
               <LoadingSearchItem variant={`circle`} />
               <LoadingSearchItem />
               <LoadingSearchItem variant={`circle`} />
             </>
           )}
-          {data && (called || profileCalled) && (
-            <SearchResults results={data} profileResults={profileData} />
+          {data && called && (
+            // @ts-ignore
```
[the remark this answers is not in the document]
Comment (austinmilt) [reply src/common/components/search/SearchBar.tsx:149]: looks like `SearchResultsProps` requires a non-undefined `results`. Do you need to make it accept undefined in the interface? (see comment below).

File: src/common/components/search/SearchResults.tsx [hunk 3]
```diff
@@ -1,17 +1,18 @@
...
+import { SearchQuery, MetadataJson, Wallet } from 'src/graphql/indexerTypes';
 import { PublicKey } from '@solana/web3.js';
 import { ProfileSearchItem, NFTSearchItem } from './SearchItems';
 import { isPublicKey } from './SearchBar';
 import { profile } from 'console';
 
 interface SearchResultsProps {
-  results: MetadataSearchQuery;
-  profileResults?: ProfileSearchQuery;
+  results: MetadataJson[];
```
Comment (austinmilt) [inline src/common/components/search/SearchResults.tsx:9]: should this be `results?: MetadataJson[];`?

**Review by austinmilt — COMMENTED** [review]
Mostly fine, but consider replacing `@ts-ignore` (see comment)

File: src/common/components/search/SearchBar.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (milorue) [reply src/common/components/search/SearchBar.tsx:149]: Yeah i did some tooling around with this i'll give it one last pass

**Review by austinmilt — APPROVED** [review]

**Pull request merged**